compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CompilerPluginRunner.java

```diff
@@ -233,6 +234,11 @@ public void visit(BLangSimpleVariable varNode) {
         notifyProcessors(attachmentList, (processor, list) -> processor.process(varNode, list));
     }
 
+    @Override
+    public void visit(BLangTupleVariable tupleVariableNode) {
+        /* ignore */
```

Comment:
As discussed with @rdhananjaya architecture of the compiler plugin runner will be changed, so that we don't add new visit methods. However, I will create an issue to track this

Reply:
#27541